src/App.js

```diff
@@ -1,23 +1,39 @@
-import logo from './logo.svg';
+import { useState } from "react";
 import './App.css';
 
 function App() {
+  const [isPlaying, setIsPlaying] = useState(false);
+  const [gameEnded, setGameEnded] = useState(false);
```

Comment:
does it need to have 2 states since playing and ended cannot have both true or false?

Reply:
yes we can use 1 state but I believe a third conditional render would be suitable for having a game over screen where a user can submit their score to the DB. I can also do a state where a user hasScore after playing the game, and a button can conditionally render to send a score.